src/Core/src/Handlers/DatePicker/DatePickerHandler.Android.cs

```diff
@@ -73,7 +78,11 @@ protected virtual DatePickerDialog CreateDatePickerDialog(int year, int month, i
 					VirtualView.Date = e.Date;
 				}
 			}, year, month, day);
-
+			if (VirtualView != null)
+			{
+				dialog.DatePicker.MaxDate = (long)VirtualView.MaximumDate.ToUniversalTimeNative().Subtract(DateTime.MinValue.AddYears(1969)).TotalMilliseconds;
+				dialog.DatePicker.MinDate = (long)VirtualView.MinimumDate.ToUniversalTimeNative().Subtract(DateTime.MinValue.AddYears(1969)).TotalMilliseconds;
+			}
```

Comment:
Well, maybe no need to the `setLater` as you do it here. In this case, we can use a totally lazy creating.